include/gsl/span

```diff
@@ -121,139 +122,165 @@ namespace details
     {
     };
 
-    template <class Span, bool IsConst>
+    template <class type>
```

Comment:
Style question: Does GSL permit lowercase template parameters like `type`?

Reply:
Fixed with Casey's comment. About to push update.